appimage/AppImageBuilder.yml

```diff
@@ -0,0 +1,65 @@
+# appimage-builder recipe see https://appimage-builder.readthedocs.io for details
+script:
+  - rm -rf AppDir | true
+  - cp -r ./build/linux/x64/{{BUILD_TYPE}}/bundle AppDir
+  - mkdir -p AppDir/usr/share/icons/hicolor/64x64/apps/
+  - cp ./assets/logo.svg AppDir/usr/share/icons/hicolor/64x64/apps/twake.svg
+version: 1
+AppDir:
+  path: ./AppDir
+  app_info:
+    id: com.linagora.linux.twake
+    name: Twake
+    icon: twake
+    version: latest
+    exec: fluffychat
```

Comment:
can not change it?

Reply:
Not until you change package name in `pubspec.yaml` to `twake`: https://github.com/linagora/twake-on-matrix/blob/04967465d5ec643d58d3e46e1ef366598b525c81/pubspec.yaml#L1

Reply:
okie. got it